### PR TITLE
Fix/acs 849 incorrect probes

### PR DESCRIPTION
**Pull request by nithinnambiar**
Add startup probe to ACS 
https://alfresco.atlassian.net/browse/ACS-849

### CODE REVIEW

File: helm/alfresco-content-services/community_values.yaml [hunk 1]
```diff
@@ -71,6 +71,10 @@ repository:
       memory: "3000Mi"
     limits:
       memory: "3000Mi"
+  # -- The startup probe to cover the worse case startup time for slow start containers
+  startupProbe:
+    periodSeconds: 30
+    failureThreshold: 20
```
Comment (mteodori) [inline helm/alfresco-content-services/community_values.yaml:77]: I would reduce to 10 and have a 5m wait for the slow container startup at most, WDYT

**Review by mteodori — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/community_values.yaml [hunk 2]
```diff
@@ -71,6 +71,10 @@ repository:
       memory: "3000Mi"
     limits:
       memory: "3000Mi"
+  # -- The startup probe to cover the worse case startup time for slow start containers
+  startupProbe:
+    periodSeconds: 30
+    failureThreshold: 10
```
Comment (alxgomz) [inline helm/alfresco-content-services/community_values.yaml:77]: see my comment in conversation.
tl;dr: I think this should not go to the main container

File: helm/alfresco-content-services/templates/deployment-repository.yaml [hunk 3]
```diff
@@ -103,6 +103,12 @@ spec:
 {{ toYaml .Values.repository.extraVolumeMounts | indent 10 }}
           {{- end }}
           {{- end }}
+          startupProbe:
+            httpGet:
+              path: /alfresco/api/-default-/public/alfresco/versions/1/probes/-ready-
+              port: {{ .Values.repository.image.internalPort }}
+            periodSeconds: {{ .Values.repository.startupProbe.periodSeconds }}
+            failureThreshold: {{ .Values.repository.startupProbe.failureThreshold }}
           readinessProbe:
```
Comment (alxgomz) [inline helm/alfresco-content-services/templates/deployment-repository.yaml:112]: see my comment in conversation.
tl;dr: I think this should not go to the main container

File: helm/alfresco-content-services/values.yaml [hunk 4]
```diff
@@ -66,7 +66,11 @@ repository:
       memory: "3000Mi"
     limits:
       memory: "3000Mi"
-  # -- The repository readiness probe is used to check startup only as a failure
+  # -- The startup probe to cover the worse case startup time for slow start containers
+  startupProbe:
+    periodSeconds: 30
+    failureThreshold: 10
+  # The repository readiness probe is used to check startup only as a failure
```
Comment (alxgomz) [inline helm/alfresco-content-services/values.yaml:73]: see my comment in conversation.
tl;dr: I think this should not go to the main container
